Application build task added

larchanka opened the pull request:
[Grunfile.js] Application build task added
[package.json] Processhtml dependency added

larchanka committed:
Gruntfile changed

Jauhen commented on Gruntfile.js:
```diff
@@ -1,132 +1,13 @@
-                'test/lib/run-qunit.js',
-                'test/*.js'
-                ]
+                files: ['Gruntfile.js', 'package.json', 'xf/src/*.js', 'xf/ui/*.js', 'js/xf.js', 'test/components/test.js', 'test/src/*.js', 'test/ui/*.js', 'test/lib/run-qunit.js', 'test/*.js']
```
don't need to flat this array

Jauhen commented on Gruntfile.js:
```diff
@@ -149,10 +28,72 @@ module.exports = function (grunt) {
+    grunt.registerTask('default', ['build', 'uglify', 'less', 'recess']);
 
-};
+}; 
```
Add new line at the end of file

Jauhen commented:
Much better to just revert previous changes then applying a new one. Technically it is the same, but keep history cleaner.

larchanka committed:
Gruntfile changed

larchanka committed:
Additional grunt task added
[Gruntfile.js] Tak for application building added
[package.json] proccesshtml dependency added

larchanka committed:
Merge pull request #3 from epam/master

Merge request
